psi4/driver/procrouting/sapt/sapt_util.py

```diff
@@ -143,21 +143,37 @@ def print_sapt_dft_summary(data, name, short=False):
     ret += "\n"
     core.set_variable("SAPT IND ENERGY", ind)
 
+
+    # Exchange-dispersion scaling
+    exch_disp_scheme = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_SCALE_SCHEME")
+    if exch_disp_scheme == "NONE":
+        data["Exch-Disp20,r"] = data["Exch-Disp20,u"] 
+    if exch_disp_scheme == "FIXED":
+        exch_disp_scale = core.get_option("SAPT", "SAPT_DFT_EXCH_DISP_FIXED_SCALE")
+        data["Exch-Disp20,r"] = exch_disp_scale * data["Exch-Disp20,u"]
+    if exch_disp_scheme == "DISP":
```

Comment:
can use elseif

Reply:
*must* use `elif` / `else` here

Reply:
I have been defined this option as an enumerate type in `read_options.cc`. Do I need to raise an error for unexpected value of `exch_disp_scheme` here? If so, what should the error message be?

Reply:
no need for `else` with error msg, just switch to `elif`.